src/main/java/tools/image/TileImageBreaker.java

```diff
@@ -132,8 +131,7 @@ private static Image loadImage() {
         tracker.waitForAll();
         return img;
       } catch (final InterruptedException e) {
-        ClientLogger.logQuietly(e);
-        return loadImage();
```

Comment:
This call was removed because it seemed wrong to recursively call into the same method after it was just interrupted.  It now simply falls through to the failure case after re-interrupting the current thread.